contents/handbook/growth/revops/credits.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Giving credits to customers
+sidebar: Handbook
+showTitle: true
+---
+
+Sometimes we might want to offer a customer one time credits to cover an upcoming invoice, for example when accommodating a trial for a new product or offering compensation for a recent incident. Here’s how to do that.
+
+- Go to [Billing Admin → Credits](https://billing.posthog.com/admin/billing/credit/)
```

Comment:
`<PrivateLink url="#">Billing Admin → Credits</PrivateLink>` will denote this with a lock icon!